build.xml

```diff
@@ -252,6 +253,15 @@
         <matches pattern="true" string="${build.isArmSoftFloat}"/>
       </and>
     </condition>
+    <condition property="jre.arch" value="mips64el">
+      <or>
+	<matches pattern="mips64el" string="${os.arch}"/>
+	<and>
+	  <matches pattern="mips64" string="${os.arch}"/>
+	  <matches pattern="little" string="${sun.cpu.endian}"/>
+	</and>
```

Comment:
Is there evidence, that a JVM reports a mips64 running in little endian mode as only `mips64`. If so there other uses of `os.arch` would also need more attention. I only had a look at openjdk on debian and there mips64el was correctly reported as `os.arch`. If there is none, I would suggest to remove the second match (lines 259-262).